docs/_build/html/_sources/tutorial.rst.txt

```diff
@@ -0,0 +1,198 @@
+####################
```

Comment:
the file extension `.rst.txt` is throwing me off. I'm used to seeing it just `.rst`. Any reason to append the `.txt` suffix?

Reply:
I honestly do not know. That's how the auto generated sphinx plugin conducted it. This will no longer be an issue once I switch to MD files